docs/_sass/sidebar.scss

```diff
@@ -130,7 +130,7 @@
         padding: 1rem 0 0 0;
         margin: 0;
         &-title a {
-          font-weight: bold;
+          font-weight: normal;
```

Comment:
@yash37158 I think it would be better, if instead of setting all title's font-weight to normal, we do just for Grandchildren. something like this: 
> <img width="217" alt="image" src="https://github.com/meshery/meshery/assets/110674407/f3dde94b-0024-4533-a33b-782d5daf15e1">

Thoughts ?

Reply:
Looks good to me, @vishalvivekm

Reply:
Yes I agree @vishalvivekm, I fix the issue